src/daemon/ca/publishing.rs

```diff
@@ -357,6 +344,24 @@ impl CaObjects {
         all_elements
     }
 
+    /// Returns the closest next update time from among manifests held by this CA
+    pub fn closest_next_update(&self) -> Option<Timestamp> {
+        let mut closest = None;
+
+        for rco in self.classes.values() {
```

Comment:
What is an "rco" ? "resource class something" ?

Reply:
I could use `resource_class_objects` it's a bit wordy but probably better

Reply:
done